doc/source/installation/digital_alliance.rst

```diff
@@ -168,6 +168,21 @@ The job is sent using:
 
   sbatch job.sh
 
+.. important::
```

Comment:
I would move this warning before line 164 (aka put the warning before the instructions on how to submit jobs. Otherwise it makes the reading of it kinda confusing. (see the image below, it's a bit weird I think)
<img width="925" height="546" alt="Screenshot 2025-08-25 at 09 44 15" src="https://github.com/user-attachments/assets/9d28c966-6ff2-437d-a859-0dcc52a7b412" />